Add gp-encoding and viewer-3d subcrates (#121)
GeoPlegma/GeoPlegma: viewer-3d/README.md

# Visualization App

> This is a simple visualization app for the GP encoding project. It allows you to visualize the results of the GP encoding library directly in a desktop application. The visualization app is built using Tauri, and it uses deck.gl for rendering the visualizations.

Reviewer: ```suggestion
This is a simple visualization app for the GP encoding project. It allows to visualize the results of the GP encoding library directly in a desktop application. The visualization app is built using Tauri, and it uses deck.gl for rendering the grids.
```

## Building and Running

To build and run the visualization app, follow these steps:

> 1. Make sure you have all the dependencies from Tauri installed on your system. You can find the prerequisites [here](https://tauri.app/start/prerequisites/).

Reviewer: ```suggestion
1. Make sure to install all the Tauri dependencies on your system. You can find the prerequisites [here](https://tauri.app/start/prerequisites/).
```
